assimilator.py

```diff
@@ -35,8 +35,8 @@ def assimilate(data_type, data_path, data_idName, geo_path, geo_idName, out_path
                 # For row in CSV insert columns as properties in geoJSON
                 for boundary in boundaries["features"]:
                     properties = boundary["properties"]
-                    #print(properties)
-                    
+                    # print(properties)
```

Comment:
Should be moved from the main folder - as well as other `.py` files.

Leaving a **general comment** here on files location restructuring - unfortunately I cannot comment on files when the change in the PR is just `move` that is why I am doing it here.

As already mentioned, the data folder should contain only (actual) _data files_ and Python modules required to read and _serve_ data to other (analysis/processing) modules.

For this reason, I would ask to move from that folder any of these files (e.g. `/data/scraping/..`, `/data/tweet-processing/..`) 

This files should live in another (specific folder) --> most likely we are going to create a unique folder containing this structure:

```
/root
    |__ covid_map
        |___ data
        |___ scrapers
        |___ tweets
        |___ api
        |___ main_script_1.py
        |___ main_script_2.py
        |___ main_server_app.py
    |__ dashboard/
    ...
    |__ docs/
    |__ README.md
```

So I would suggest start re-organising the whole folder this way.. WDYT?

Reply:
I am following this structure now :)

Reply:
@leriomaggio Does `API/` here refer to all of the borg related files? I.e. the ones moved out of `dashboard/` and `data/` ?  Just want to make sure I'm putting them in the right place!

Reply:
Implemented in latest commits :)